### PR TITLE
Task/bws20 master-sign-in

### PR DESCRIPTION
**Pull request by sofia0602**

### CODE REVIEW

File: shared/src/validation/master/master-sign-in/master-sign-in.validation-schema.ts [hunk 1]
```diff
@@ -1,19 +1,39 @@
 import * as Joi from 'joi';
 import { getNameOf } from '~/helpers/helpers';
 import { MasterSignInRequestDto } from '~/common/types/types';
-import { MasterValidationMessage } from '~/common/enums/enums';
+import {
+  MasterValidationMessage,
+  MasterSignInValidationRule,
```
Comment (what1s1ove) [inline shared/src/validation/master/master-sign-in/master-sign-in.validation-schema.ts]:
```suggestion
  MasterValidationRule,
```

File: shared/src/common/enums/enums.ts [hunk 2]
```diff
@@ -5,5 +5,6 @@ export { HttpCode, HttpHeader, HttpMethod } from './http/http';
 export {
   MasterValidationMessage,
   MasterSignUpValidationRule,
```
Comment (what1s1ove) [inline shared/src/common/enums/enums.ts]: MasterSignUpValidationRule rename to `EAMMasterValidationRule` and keep all rules (sign-in and sign-up) in one place

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: shared/src/common/enums/enums.ts (hunk 2 above)
Comment (Yaroslaww-1) [reply shared/src/common/enums/enums.ts]: *`EAMMasterValidationRule`

**Review by what1s1ove — DISMISSED** [review]

File: backend/src/validation-schemas/master/master.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export {
```
Comment (Yaroslaww-1) [inline backend/src/validation-schemas/master/master.ts]: File name should be `backend/src/validation-schemas/eam-master/eam-master.ts`

File: backend/src/validation-schemas/validation-schemas.ts [hunk 4]
```diff
@@ -1 +1 @@
-export { eamMasterSignUp, eamMasterSignIn } from './eam-master/eam-master';
+export { eamMasterSignUp, eamMasterSignIn } from './master/master';
```
Comment (Yaroslaww-1) [inline backend/src/validation-schemas/validation-schemas.ts]: `export { eamMasterSignUp, eamMasterSignIn } from './eam-master/eam-master';`

**Review by Yaroslaww-1 — CHANGES_REQUESTED** [review]

File: shared/src/validation/eam-master/eam-master-sign-in/eam-master-sign-up.validation-schema copy.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import * as Joi from 'joi';
```
Comment (what1s1ove) [inline shared/src/validation/eam-master/eam-master-sign-in/eam-master-sign-up.validation-schema copy.ts:1]: is it duplicate ? 🤔

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Pull request closed without merging**